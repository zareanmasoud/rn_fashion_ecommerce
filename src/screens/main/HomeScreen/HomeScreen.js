import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import HomeHeader from 'features/home/layout/Header';
// eslint-disable-next-line import/no-extraneous-dependencies
import ProductList from 'features/home/ProductList';
// eslint-disable-next-line import/no-extraneous-dependencies
import MainTabbar from 'features/home/layout/MainTabbar';
// import styles from './styles';

export default function HomeScreen() {
  return (
    <>
      <ProductList />
      <MainTabbar />
    </>
  );
}

HomeScreen.navigationOptions = () => {
  return {
    header: <HomeHeader />,
  };
};
