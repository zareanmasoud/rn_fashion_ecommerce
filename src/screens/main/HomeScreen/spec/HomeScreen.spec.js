import React from 'react';
import {shallow} from 'enzyme';
import HomeScreen from '../HomeScreen';

describe('HomeScreen', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(<HomeScreen />);
      expect(component).toMatchSnapshot();
    });
  });
});
