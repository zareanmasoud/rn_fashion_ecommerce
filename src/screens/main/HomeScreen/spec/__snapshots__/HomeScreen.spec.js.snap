// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen Rendering should match to snapshot 1`] = `
<ScreenContentFrame
  statusBackgroundColor="#ffffff"
  statusBarStyle="dark-content"
>
  <HomeHeader />
  <Text
    testID="View"
  />
  <ProductList />
  <withNavigation(HomeTabbar) />
</ScreenContentFrame>
`;
